media/css/externalpages/pocket/components/_platform-nav.scss

```diff
@@ -0,0 +1,99 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+
+.pocket-header.platform {
+    box-shadow: 0 0 5px rgba(0, 0, 0, .15);
+    border-bottom-color: #c8c8c8;
```

Comment:
Same here with the hexcode as I mentioned above, let's find a nice token from Protocol to use instead! It helps with consistency :)

Reply:
Yeah - worry is changing what the hex is on the pocket site without approval on changing the color. Thought was to get it up and running and when the redesign happens we can make an actual pocket theme - since none of the colors in protocol are actually _for_ pocket.